typescript/lib/constructor_calls.ts

```diff
@@ -0,0 +1,219 @@
+import { PartitionID } from './networks/network_context';
+import { LambdaID } from "./modules/lambda_context";
+
+// // ====================================================================================================
+// //                                    Service Context
+// // ====================================================================================================
+
+export function newExecCommandArgs(serviceId: ServiceID, command: string[]): ExecCommandArgs {
+    const result: ExecCommandArgs = new ExecCommandArgs();
+    result.setServiceId(serviceId);
+    result.setCommandArgsList(command);
+
+    return result;
+}
+
+export function newGenerateFilesArgs(serviceId: ServiceID, fileGenerationOpts: Map<string, FileGenerationOptions>): GenerateFilesArgs {
+    const result: GenerateFilesArgs = new GenerateFilesArgs();
+    result.setServiceId(String(serviceId)); 
+    for (let fileID in fileGenerationOpts) {
+        result.getFilesToGenerateMap().set(fileID, fileGenerationOpts[fileID]);
+    }
+    
+    return result;
+}
+
+export function newFileGenerationOptions(filesToGenerateSet: Set<string>): Map<string, FileGenerationOptions> {
+    const result: Map<string, FileGenerationOptions> = new Map();
+    for (let fileId in filesToGenerateSet) {
+        result[fileId] = new FileGenerationOptions();
+        result[fileId].setFileTypeToGenerate(FileGenerationOptions.FileTypeToGenerate.FILE);
+    }
+
+    return result;
+}
+
+export function newLoadStaticFilesArgs(serviceId: ServiceID, staticFilesToCopyStringSet: Map<string, boolean>): LoadStaticFilesArgs {
+    const result: LoadStaticFilesArgs = new LoadStaticFilesArgs();
+    result.setServiceId(String(serviceId));
+    const staticFilesMap: Map<string, boolean> = result.getStaticFilesMap();
+    for (let staticFildID in staticFilesToCopyStringSet) {
+        staticFilesMap.set(staticFildID, staticFilesToCopyStringSet[staticFildID]);
+    }
+
+    return result;
+}
+
+// // ====================================================================================================
+// //                                    Network Context
+// // ====================================================================================================
+
+export function newLoadLambdaArgs(lambdaId: LambdaID, image: string, serializedParams: string): LoadLambdaArgs {
+    const result: LoadLambdaArgs = new LoadLambdaArgs();
+    result.setLambdaId(String(lambdaId));
+    result.setContainerImage(image);
+    result.setSerializedParams(serializedParams);
+
+    return result;
+}
+
+export function newRegisterStaticFilesArgs(strSet: Map<string, boolean>): RegisterStaticFilesArgs {
+    const result: RegisterStaticFilesArgs = new RegisterStaticFilesArgs();
+    const staticFilesSetMap: Map<string, boolean> = result.getStaticFilesSetMap();
+    for (let staticFileID in strSet) {
+        staticFilesSetMap.set(staticFileID, strSet[staticFileID]);
```

Comment:
if we were to use the value `strSet[staticFileID]` then we should iterate over the map via `Map.entries()`. However, this value actually doesn't matter so you should just use `true` (to emphasize that this is a set, rather than a map that contains actual values). You should do this for all other set-like Maps in this file as well.